app/ws/connectionMappings.ts

```diff
@@ -0,0 +1,61 @@
+import redis from '../redis/redis';
+import { acquireLock } from './channelLocks';
+
+const channelListeners: Map<string, Set<string>> = new Map();
+const connectionToChannels: Map<string, Set<string>> = new Map();
+const userConnections: Map<string, Set<string>> = new Map();
+
```

Comment:
add comments explaining keys and values for each map

Reply:
![image](https://github.com/user-attachments/assets/36630e8e-0d13-4581-ae0e-abe44d09f483)